tests/plugin/validate.proto

```diff
@@ -8,7 +8,7 @@ import "validate/validate.proto";
 option go_package = "github.com/alta/protopatch/tests/plugin";
 
 message Interface {
```

Comment:
This should probably have a completely separate test message(s) for testing type casts, rather than test both the validation plugin and type casts in the same message.